source/scss/components/_hero-image.scss

```diff
@@ -0,0 +1,21 @@
+.heroImageContainer {
+  height: 200px;
+  overflow: hidden;
+  position: relative;
+  text-align: center;
+  width: 100%;
+
+  img {
+    margin-left: -9999px;
+    margin-right: -9999px;
+    min-height: 100%;
+    min-width: 100%;
+    position: relative;
+  }
```

Comment:
The image looks like it is always 'anchored' to the top. Is this the behaviour they want? The image on Urlaubspiraten behaves differently.

Reply:
I mean, when we resize the page, the `<img>` tag gives priority to always display the top part of the image file, instead of just hiding its left and right content.

Reply:
When the page is too wide for the image, we could instead make the image display its central content instead of anchoring it to its top. What do you say?

Reply:
You're right. looking again...

Reply:
Ok! If you need help with that, let me know.

Reply:
done

Reply:
It looks much better than before, but I still see it 'anchored' to the top on very wide screens:
![image](https://user-images.githubusercontent.com/45654887/62920633-a01d3400-bda6-11e9-93bd-a09926c0b295.png)
But since it's highly unlikely that someone has a screen with those proportions, I say this is done, and it looks great!